sdk/core/Azure.Core.TestFramework/src/RecordedTestBaseOfTEnvironment.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace Azure.Core.TestFramework
 {
+    [NonParallelizable]
```

Comment:
This can probably go on RecordedTestBase.

Reply:
If the problem you mentioned is actually because of event trace testing, shouldn't we put this on ClientTestBase?

Reply:
No, I don't think so since there are unit tests that derive from ClientTestBase.

Reply:
Good point. I even do that. 🤦‍♂️